### ISSUE TITLE
way to bookmark images

### ISSUE THREAD

**Issue by noc76** [Chirmaya/BooruSlideshow#47]
says in title, would be convenient

**Comment by Chirmaya** [comment]
Agreed! I was thinking of some sort of playlist or way to go through all the ones you liked. This will likely be the next update.
